doc/tutorials/walkthrough_inputs.rst

```diff
@@ -353,13 +353,19 @@ Now we need to specify some **settings** that define fundamental reactor
 parameters, as well as modeling approximation options. For this, we make a
 **settings file**, called ``anl-afci-177.yaml``.
 
-First we need to supply some metadata.
+First we need to supply some metadata about the settings themselves:
 
 .. literalinclude:: ../../armi/tests/tutorials/anl-afci-177.yaml
     :language: yaml
     :start-after: begin-metadata
     :end-before: begin-settings
 
+Here, the version is "uncontrolled" because this is just a test file we want
+to always work with the latest ARMI. But if you are doing a really important
+analysis, you probably want to specify which version of ARMI you used, like:
+"version: 0.2.5", or similar. Obviously, metadata is for documentation
+purposes, so it's optional. And "uncontrolled" is the preferred default.
```

Comment:
I think it is actually _not_ optional.

Reply:
I believe that ARMI will not be able to load a settings file if this is not in it.

Reply:
Yeah, I was being a little loosey-goosey with my language, that's a good call.

What I was _thinking_ was specifying a real version was "optional"; as opposed to just specifying it as "uncontrolled".

I'll fix it.

Reply:
@keckler @jakehader Is that better?